ruby_on_rails/rspec.md

```diff
@@ -79,6 +84,27 @@ end
 
 to use headless chrome for system tests.
 
+### Optional: catch javascript errors
+
+If you want to catch Javascript errors in your system tests, you can add the following to the `rails_helper.rb`:
+
+```ruby
+config.after(:each, type: :system, js: true) do
```

Comment:
🕺

Reply:
by the way there is quite a nice article of a guy you might never heard of:
https://medium.com/@coorasse/catch-javascript-errors-in-your-system-tests-89c2fe6773b1